package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zimbra/zimlet-cli",
-  "version": "8.3.0",
+  "version": "8.3.1",
```

Comment:
revert this, in zimlet-cli we don't bump version in PR

Reply:
done